compiler/lexer.nim

```diff
@@ -733,11 +780,11 @@ proc getEscapedChar(L: var Lexer, tok: var Token) =
 
 proc handleCRLF(L: var Lexer, pos: int): int =
   template registerLine =
-    let col = L.getColNumber(pos)
-
     when not defined(nimpretty):
+      let col = L.getColNumber(pos)
       if col > MaxLineLength:
         lexMessagePos(L, hintLineTooLong, pos)
+    discard
```

Comment:
```suggestion
```

(`when` is sufficient)

Reply:
comment still holds

Reply:
Sorry; I'm hitting "resolve conversation" as I'm editing the code prior to commit. :) You should see it now.

Reply:
oh ya if figured. IIRC there's a way to batch replies, by clickign on "start a review" instead of "add a single comment" and then upon commit, click send review so they all appear at same time after commit. But either way is fine.